printer-linter/src/printerlinter/formatters/def_json_formatter.py

```diff
@@ -1,29 +1,84 @@
 import json
 import re
+from collections import OrderedDict
 from pathlib import Path
+from typing import Dict
 
 from .formatter import FileFormatter
 
+
+# Dictionary items with matching keys will be sorted as if they were the value
+# Example: "version" will be sorted as if it was "0"
+TOP_LEVEL_SORT_PRIORITY = {
+    "version":  "0",
+    "name":     "1",
+    "inherits": "3"
+}
+
+METADATA_SORT_PRIORITY = {
+    "visible": "0",
+    "author": "1",
+    "manufacturer": "2",
+    "file_formats": "3",
+    "platform" : "4"
```

Comment:
If you decide to align the `TOP_LEVEL_SORT_PRIORITY` also apply that alignment in the dict below.
```suggestion
    "visible":      "0",
    "author":       "1",
    "manufacturer": "2",
    "file_formats": "3",
    "platform" :    "4",
```
Also added a trailing comment on the last entry, to ensure that any future diff will not change two lines